cloudinit/net/openbsd.py

```diff
@@ -0,0 +1,41 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+
+from cloudinit import log as logging
+from cloudinit import util
+import cloudinit.net.bsd
+
+LOG = logging.getLogger(__name__)
+
+
+class Renderer(cloudinit.net.bsd.BSDRenderer):
+
+    def write_config(self):
+        for device_name, v in self.interface_configurations.items():
+            if_file = 'etc/hostname.{}'.format(device_name)
+            fn = util.target_path(self.target, if_file)
+            content = 'dhcp\n'
+            try:
+                if isinstance(v, dict):
+                    content = "inet {address} {netmask}\n".format(
+                                address=v['address'],
+                                netmask=v['netmask'])
+            except KeyError:
+                LOG.error("Invalid static configuration for %s", device_name)
```

Comment:
Should the log message mention that we're falling back to DHCP?

Reply:
I thought about it too, but I think it will obvious enough.

Reply:
I think it will be obvious that a system is using DHCP; I'm not so sure it will be obvious _why_ if people can't search "dhcp" in the log and find this line.